keyboards/crkbd/keymaps/cameronjlarsen/features/oneshot.c

```diff
@@ -0,0 +1,51 @@
+#include "oneshot.h"
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
I've added a GPL2 license header to all files missing them